Project.toml

```diff
@@ -94,7 +94,7 @@ PrecompileTools = "1.1"
 Preferences = "1.3"
 Printf = "1"
 RecipesBase = "1.1"
-RecursiveArrayTools = "2.38.10"
+RecursiveArrayTools = "3.27.1"
```

Comment:
I'm curious what’s stopping it from updating to the latest version?

Reply:
This is actually for the Downgrade.yml CI. It tests CI using the lower bound on a package; we would need 3.27.1 since that's where your PR was released.